helpers/http_client_test.go

```diff
@@ -1,35 +1,47 @@
-package helpers
+package helpers_test
 
 import (
 	"io/ioutil"
 	"os"
 	"testing"
 
+	"github.com/ory/hydra-maester/helpers"
+
 	"github.com/stretchr/testify/require"
 )
 
 func TestCreateHttpClient(t *testing.T) {
 	t.Run("should create insecureSkipVerify client", func(t *testing.T) {
-		client := CreateHttpClient(true, "")
+		client, err := helpers.CreateHttpClient(true, "")
 		require.NotNil(t, client)
+		require.Nil(t, err)
 	})
 
 	t.Run("should create client with insecureSkipVerify and wrong tlsTrustStore", func(t *testing.T) {
 		tlsTrustStore := "some path"
-		client := CreateHttpClient(true, tlsTrustStore)
+		client, err := helpers.CreateHttpClient(true, tlsTrustStore)
 		require.Nil(t, client)
+		require.Nil(t, err)
 	})
 
 	t.Run("should create client with and tlsTrustStore", func(t *testing.T) {
 		file, err := ioutil.TempFile("/tmp", "test")
 		require.Nil(t, err)
-		client := CreateHttpClient(true, file.Name())
+		client, err := helpers.CreateHttpClient(true, file.Name())
 		defer os.Remove(file.Name())
 		require.NotNil(t, client)
+		require.Nil(t, err)
+	})
+
+	t.Run("should not create client with and wrong tlsTrustStore", func(t *testing.T) {
+		client, err := helpers.CreateHttpClient(true, "/somefile")
+		require.NotNil(t, client)
+		require.NotNil(t, err)
```

Comment:
Since we control what error will be returned here, maybe we could check not only if the err is not nil, but if it the one we expect?